docs/logs/correlation/Program.cs

```diff
@@ -7,39 +7,32 @@
 using OpenTelemetry.Logs;
 using OpenTelemetry.Trace;
 
-namespace Correlation;
+ActivitySource activitySource = new("MyCompany.MyProduct.MyLibrary");
 
-public class Program
+using var loggerFactory = LoggerFactory.Create(builder =>
```

Comment:
from my experience, a lot of people accidentally end up copying this as-is, including the "using" keyword. I recommend to use explicit Dispose() call at the end of the application.

The [getting started](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-console/Program.cs#L27-L29) uses such pattern already, would be good to stick with that throughout the docs.

Reply:
I considered this and my guess is that most folks would copy code from the getting-started docs instead of this one, and this doc is more focusing on correlation.
Do you think all the examples should use explicit dispose?

Reply:
I've made the updates by using explicit dispose.

There are still two things I'm not happy with, any suggestions?
1. The ActivitySource is not static readonly (if I need to make it static readonly, I have to split the example into 2 *.cs files and add some junk code).
2. The name of ActivitySource is not following the fully qualified type name.

Reply:
Can we steal from https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-console/Program.cs#L8-L15

Reply:
As I explained originally, this requires a class to be explicitly defined, which forces the custom log method to move to a separate file, we will end up with a much more complex project with multiple *.cs files. Do you consider it better?